### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by ptomato**
Gets linting working for the JSX code, and makes the definition of global variables more explicit as I outlined in https://github.com/sarahghp/decimal-playground/pull/46#discussion_r936097042. I'm curious what you think about the last two commits, we can keep both, one, or none of them.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -67,8 +67,10 @@ const UNPATCHED_MATH_METHODS = Object.getOwnPropertyNames(Math).filter((el) => {
 });
 
 const SHARED_SINGLE_OPS = {
+  "+": "add",
   "*": "mul",
   "-": "sub",
+  "/": "div",
```
Comment (sarahghp) [inline src/constants.js:73]: This causes problems with `div` throwing in Big Decimal mode — e.g., it means it does not. Livewise, in the transforms `add` is not necessary becase it is transformed into `addOrConcat`. Let's not change it here, but you can import the base levels and then `add` and `div` to the spread object like we do in `Decimal128`.

**Review by sarahghp — CHANGES_REQUESTED** [review]
Honestly I like the top-level solution. The only thing I dislike is wrapping all the code inside one function — I find that working one indentation in when not needed makes files slightly more annoying to work with. In this case, we can move the function definitions outside `defineGlobals` and just leave the `Object.assign` inside with the same effect, right?

So if you fix the "dedupe" problem (it's not really a meaningless duplication) and move the definitions out, this can go.

File: src/constants.js (hunk 1 above)
Comment (ptomato) [reply src/constants.js:73]: Oops, I thought it was just a duplication that had gotten out of sync. (Well, I guess that's why we should get the tests working again :stuck_out_tongue:) I didn't mean to open another can of worms, sorry.

**Review by sarahghp — APPROVED** [review]
LGTM thanks.

**Pull request merged**